lib/index.js

```diff
@@ -49,6 +51,7 @@ exports.register = (server, options, next) => {
       headers['X-Rate-Limit-Remaining'] = rate.remaining;
       headers['X-Rate-Limit-Limit'] = rate.limit;
       headers['X-Rate-Limit-Reset'] = rate.reset;
+      headers['X-Rate-Limit-Window'] = rate.window;
```

Comment:
What do you mean pass down this value? I can't find anything online that says this is a standard header for rate limiting.